### DISCUSSION TITLE
Feedback on 1C (Cross-validation Introduction)

### DISCUSSION THREAD

**Discussion by jmgirard** [PittMethods/appliedml#21] [Ideas]
Slide 3:
- Be sure to briefly restate what we mean by "overfitting" and "generalizability" since this 1C will be a while after 1A

Slide 5: 
- Be sure to define "data generating process" and "random noise" in this context
- Be sure to clearly define "leaking information" and give a concrete example (consider adding to slide)

Slide 11:
- Be sure to describe which variables would be used as labels and features in this dataset
- To make things simpler for attendees, consider having two chunks for reading the data: one that is eval=TRUE but echo=FALSE that calls `"../data/affect.csv"` (to actually work on your computer but hidden) and one that is eval=FALSE and echo=TRUE that calls `"affect.csv"` (to show attendees how to do it on their own computers) as the latter is what most attendees will have

Slide 21:
- Can we include the function that does multilevel/clustered resampling?

Slide 23:
- Should we adopt the tidymodels language and refer to k-fold CV as v-fold CV?

**Comment by jmgirard** [comment]
Also, consider adding some "comprehension check" quizzes (ideally 2 to 4). Here is one I had related to cross-validation in the 2021 version of 1A you could adapt.

https://github.com/PittMethods/appliedml/blob/cf120ed43cfed2e886a034cbcfcf96813a338a8b/Day_1A/Day_1A_Slides.Rmd#L1024-L1051

**Reply by ShirleyBWang** [reply]
Awesome - thanks so much for all these suggestions. Made the edits and pushed newest version! Just waiting for your `group_initial_split()` function to add in and then it'll be all set + finalized. (No worries if you can't get around to it - I can just share `group_vfold_cv()` as well!)
